systems/controllers/CMakeLists.txt

```diff
@@ -41,13 +47,12 @@ if (MATLAB_FOUND)
       pods_use_pkg_config_packages(instantaneousQPControllermex gurobi)
 
       add_mex(constructQPDataPointerMex constructQPDataPointerMex.cpp)
-      target_link_libraries(constructQPDataPointerMex drakeQP drakeControlUtil)
-      pods_use_pkg_config_packages(constructQPDataPointerMex gurobi)
+      target_link_libraries(constructQPDataPointerMex drakeQP drakeControlUtil drakeQPCommon)
+      pods_use_pkg_config_packages(constructQPDataPointerMex gurobi drake-qp-common)
```

Comment:
i wouldn't think you'd need this??  in fact, it seems risky, since the pkg config file would be written presumably after the cmake processing.